src/scripts/components/Card.js

```diff
@@ -1,58 +1,101 @@
+/* eslint-disable import/no-cycle */
 import '@styles/Card.scss';
 
+import Modals from '@components/Modals';
+
+import Dashboard from '@components/Dashboard';
+import CardService from '@services/CardService';
+import emitter from '@services/EventEmitter';
+
 export default class Card {
-  constructor(id, title, description, createdAt) {
+  constructor({
+    id,
+    title,
+    status,
+    description,
+    createdAt,
+  }) {
     this.id = id;
     this.title = title;
+    this.status = status;
     this.description = description;
     this.createdAt = createdAt;
 
-    this.editCard = this.editCard.bind(this);
-    this.deleteCard = this.deleteCard.bind(this);
+    this.edit = this.edit.bind(this);
+    this.delete = this.delete.bind(this);
 
     this.convertDate = this.convertDate.bind(this);
+    this.render = this.render.bind(this);
     this.buildView = this.buildView.bind(this);
     this.registerListener = this.registerListener.bind(this);
-
-    this.registerListener();
   }
 
-  editCard(e) {
-    console.log(e);
+  async edit() {
```

Comment:
Can we add card status in editing so that in the select it would by default show cards status?

Reply:
I has been trying to find something about this in the official documentation.

Reply:
Yes, we can) I fix it.

Reply:
Great!)